crafters/nlp/PDFExtractor/tests/test_pdfextractor.py

```diff
@@ -1,15 +1,20 @@
 from .. import PDFTextExtractor
 
 
+expected_text = "A cat poem\nI love cats, I love every kind of cat,\nI just wanna hug all of them, but I can't," \
```

Comment:
Perfect, how does it then behave with pdfs that are in chinese?

Reply:
Looks like it's not encoding those characters, they are being ignored at the moment